src/core/player.js

```diff
@@ -29,16 +29,30 @@ DM.provide('',
 {
     player: function(element, options)
     {
+        element = DM.$(element);
+        if (DM.Player._INSTANCES[element.id] !== undefined)
```

Comment:
not sure we want to test this because Players of the page added with `<iframe>` are not in `DM.Player._INSTANCES`